src/shared/GlobalTopBanners/TokenlessBanner/TokenlessBanner.tsx

```diff
@@ -0,0 +1,160 @@
+          data-testid="token-trigger"
+        >
+          the token.
+        </Tooltip.Trigger>
+        <Tooltip.Portal>
+          <Tooltip.Content
+            className="bg-ds-gray-primary p-3 text-sm text-ds-gray-octonary"
+            side="bottom"
+          >
+            <div className="flex items-center justify-center">
+              {formattedToken}
+              <div className="flex">
+                <Button
+                  variant="plain"
+                  onClick={() => setIsTokenVisible(!isTokenVisible)}
+                  to={undefined}
+                  disabled={undefined}
+                  hook="hide-and-show-org-token"
+                >
+                  <Icon
+                    name={isTokenVisible ? 'eyeOff' : 'eye'}
+                    className="text-ds-blue-darker"
+                    data-testid={isTokenVisible ? 'hide-token' : 'show-token'}
+                  />
+                </Button>
+                <CopyClipboard
+                  value={orgUploadToken}
+                  data-testid="clipboard-copy-token"
+                />
+              </div>
+            </div>
+            <Tooltip.Arrow className="size-4 fill-ds-gray-primary" />
+          </Tooltip.Content>
+        </Tooltip.Portal>
+      </Tooltip.Root>
+    </Tooltip>
+  )
+}
+
+function AdminTokenlessBanner() {
+  const { provider, owner } = useParams<UseOrgUploadTokenParams>()
+  const { data: orgUploadToken } = useOrgUploadToken({
+    provider,
+    owner,
+  })
+
+  return (
+    <TopBanner>
+      <TopBanner.Start>
+        <p className="items-center gap-1 md:flex">
+          <span className="flex items-center gap-1 font-semibold">
+            <Icon name="informationCircle" />
+            Uploading with token is now required.
+          </span>
+          You must upload with{' '}
+          {isString(orgUploadToken) ? (
+            <OrgUploadTokenTooltip orgUploadToken={orgUploadToken} />
+          ) : (
+            'the token. '
+          )}
+          Admins can manage the
+          <A
+            to={{
+              pageName: 'orgUploadToken',
+            }}
+            hook="org-upload-token-link"
+            isExternal={false}
+          >
+            global upload token settings.
+          </A>
+        </p>
+      </TopBanner.Start>
+      <TopBanner.End>
+        <TopBanner.DismissButton>Dismiss</TopBanner.DismissButton>
+      </TopBanner.End>
+    </TopBanner>
+  )
+}
+
+function MemberTokenlessBanner() {
+  const { provider, owner } = useParams<UseOrgUploadTokenParams>()
+  const { data: orgUploadToken } = useOrgUploadToken({
+    provider,
+    owner,
+  })
+
+  return (
+    <TopBanner>
+      <TopBanner.Start>
+        <p className="items-center gap-1 md:flex">
+          <span className="flex items-center gap-1 font-semibold">
+            <Icon name="informationCircle" />
+            Uploading with token is now required.
```

Comment:
nit: with "A" token?

Reply:
or "The" actually to stay in line with the below copy

Reply:
https://www.figma.com/design/CRM2ODN4ihBWITjC3xryF1/GH-2100?node-id=2001-11&node-type=frame&t=a6FXMXCubyJ4NJxJ-0 your suggestion makes more sense though!